packages/css/src/components/top-task-link/top-task-link.scss

```diff
@@ -32,6 +34,7 @@
     line-height: var(--amsterdam-top-task-link-label-compact-line-height);
   }
 
+  @include hyphenation;
```

Comment:
Top task Link has text level 4. Shouldn't we hyphenate all text level 4 elements if Top task needs to be hyphenated?